vm/src/obj/objint.rs

```diff
@@ -735,6 +723,26 @@ fn int_new(cls: PyClassRef, options: IntOptions, vm: &VirtualMachine) -> PyResul
     PyInt::new(options.get_int_value(vm)?).into_ref_with_type(vm, cls)
 }
 
+#[derive(FromArgs)]
+struct IntFromByteOptions {
```

Comment:
Suggestion here to rename this struct into `IntFromByteArguments`. Likewise for the other struct below `IntToByteArguments`.

Reply:
Thank you for reviewing. I change struct name. And also change argument name from `option` to `args`